docs/source/conf.py

```diff
@@ -307,8 +307,7 @@
     "kedro.pipeline",
     "kedro.runner",
     "kedro.config",
-    "kedro.extras.datasets",
```

Comment:
If I understand correctly, `kedro.extras.datasets` is disappearing in Kedro 0.19, so this deletion is okay